Backend/PersonDetection.Backend.Application/Services/Implementations/AuthorizationService.cs

```diff
@@ -0,0 +1,60 @@
+using FluentValidation;
+using FluentValidation.Results;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using PersonDetection.Backend.Application.Common.Exceptions;
+using PersonDetection.Backend.Application.Common.Models.Requests.Login;
+using PersonDetection.Backend.Application.Common.Models.Requests.Register;
+
+namespace PersonDetection.Backend.Application.Services.Implementations;
+
+public class AuthorizationService(
+    UserManager<IdentityUser> _userManager,
+    SignInManager<IdentityUser> _signInManager) : IAuthorizationService
+{
+    public async Task<IResult> RegisterAsync(RegisterRequest registerRequest, IValidator<RegisterRequest> validator)
+    {
+        await ValidateModelAsync(registerRequest, validator);
```

Comment:
The backend consists of only 5 endpoints, so I've decided to leave the validation as it is, instead of using attributes or reflection.